src/configure/configure.ts

```diff
@@ -197,15 +242,14 @@ class PipelineConfigurer {
 
     private async getAzureResourceDetails(): Promise<void> {
         // show available subscriptions and get the chosen one
-        let subscriptionList = extensionVariables.azureAccountExtensionApi.subscriptions.map((subscriptionObject) => {
+        let subscriptionList = extensionVariables.azureAccountExtensionApi.filters.map((subscriptionObject) => {
```

Comment:
What is filters property ?

Reply:
Azure account extension provides a way to filter out the subscriptions that you want to work on. Try using command Select subscriptions. This is similar to the subscriptions filter available in azure portal at top right, it means your context is set to use these subscriptions only. We wanted to show only the selected subscriptions set by user to work with rather than using all the subscriptions in dropdown